### PR TITLE
New camera parallax lesson

### PR DESCRIPTION
**Pull request by ClementPasteau**

### CODE REVIEW

File: tutorials/in-app/cameraParallax.json [hunk 1]
```diff
@@ -0,0 +1,750 @@
...
+        "standalone": true,
+        "description": {
+          "messageByLocale": {
+            "en": "This game is a simple platformer.\nYou can move the player with the arrow keys and jump with space.\n\n**Test it out** with the **preview** button.\n\nWhen you're ready, click on this button to continue.",
+            "fr": "Ce jeu est un simple jeu de plateforme.\nVous pouvez déplacer le joueur avec les flèches et sauter avec Espace.\n\n**Testez-le** avec le bouton **Aperçu**.\n\nQuand vous êtes prêt, cliquez sur ce bouton pour continuer.",
+            "es": "Este juego es un simple plataformero.\nPuedes mover al jugador con las flechas y saltar con la barra espaciadora.\n\n**Pruebalo** con el botón **Previsualizar**.\n\nCuando estés listo, haz clic en este botón para continuar.",
+            "pt": "Este jogo é um simples plataforma.\nVocê pode mover o jogador com as setas e pular com espaço.\n\n**Teste-o** com o botão **pré-visualização**.\n\nQuando estiver pronto, clique neste botão para continuar."
+          }
+        },
+        "placement": "right"
```
Comment (AlexandreSi) [inline tutorials/in-app/cameraParallax.json:104]: To remove, it's a bit weird imho
image

Comment (AlexandreSi) [reply tutorials/in-app/cameraParallax.json:104]: I think it displays on top by default

File: tutorials/in-app/cameraParallax.json [hunk 2]
```diff
@@ -0,0 +1,750 @@
...
+    },
+    {
+      "elementToHighlightId": "#parameter-2-expression-field",
+      "nextStepTrigger": {
+        "valueEquals": "CameraCenterX()/8"
+      },
+      "tooltip": {
+        "description": {
+          "messageByLocale": {
+            "en": "Let's make the far background move way **slower** than the camera,\n\n Enter `CameraCenterX()/8`.",
```
Comment (AlexandreSi) [inline tutorials/in-app/cameraParallax.json:602]: I would suggest to double/triple click the code, it's hard to select

File: tutorials/in-app/cameraParallax.json [hunk 3]
```diff
@@ -0,0 +1,750 @@
...
+            "es": "Ya terminé",
+            "pt": "Terminei"
+          }
+        }
+      },
+      "tooltip": {
+        "standalone": true,
+        "description": {
+          "messageByLocale": {
+            "en": "See the **black square** in the middle of the scene? This is the **camera view**. It's the point of view from which the player will see the game.\n\nLet's resize the **$(farBackground)** object so that it covers the whole black square\n\nWhen you're done, click below.",
```
Comment (AlexandreSi) [inline tutorials/in-app/cameraParallax.json:458]: You could add a base64 image to show how to do it!

Comment (ClementPasteau) [reply tutorials/in-app/cameraParallax.json:458]: added! nice addition indeed

File: tutorials/in-app/cameraParallax.json (hunk 1 above)
Comment (ClementPasteau) [reply tutorials/in-app/cameraParallax.json:104]: yes better indeed

**Pull request merged**